salesforce_bulk/salesforce_bulk.py

```diff
@@ -174,11 +174,11 @@ def create_delete_job(self, object_name, **kwargs):
         return self.create_job(object_name, "delete", **kwargs)
 
     def create_job(self, object_name=None, operation=None, contentType='CSV',
-                   concurrency=None, external_id_name=None, pk_chunking=False):
+                   concurrency=None, external_id_name=None, pk_chunking=False,
+                   extra_headers={}):
```

Comment:
This should default to None and be detected for and initialized to empty dict in the method body or else you will get unexpected results. See https://stackoverflow.com/questions/1132941/least-astonishment-and-the-mutable-default-argument